data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/PathUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.scalar.db.dataloader.core.util;
+
+public class PathUtil {
+
+  /**
+   * Ensures the specified path has a trailing slash.
+   *
+   * <p>java.nio.file.Path is not used because this is also used for virtual paths.
+   *
+   * @param path the path
+   * @return the path with a trailing slash
+   */
+  public static String ensureTrailingSlash(String path) {
+    if (path == null || path.isEmpty()) {
+      return "";
+    }
+
+    if (!path.endsWith("/")) {
```

Comment:
[minor] If this tool could be used on Windows environment, `java.io.File#separatorChar` should be used instead.

Reply:
I have updated the code to use the `java.io.File#separatorChar`.
Thank you.